### DISCUSSION TITLE
What comes next in this sequence ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#67] [Galaxy Qs] [answered]
`1   4   8   13   21   30   36  ?`

**Comment by mitchiemt11** [comment]
@Sriansh-raj My first  approach was to use the Fibonacci sequence but, I quickly realised that this set doesn't seem to resemble the Fibonacci sequence in any significant way. 😄 😄 

So considering the differences between terms, the next term can be **43**. Here is the break down:

- By calculating the first set of differences between consecutive terms in the sequence:

`4−1 =3`
`8−4=4`
`13−8=5`
`21−13=8`
`30−21=9`
`36−30=6`

This gives the differences:  `[3,4,5,8,9,6].`

- Next, we look at the differences between these differences (the second set of differences):

`4−3=1`
`5−4=1`
`8−5=3`
`9−8=1`
`6−9=−3`
​
  
This gives us a second set of differences: `[1,1,3,1,−3]`


- Looking at the second set of differences, we can attempt to discern a pattern. It seems to go: `[1,1,3,1,−3]`

This might suggest a pattern: increase by 1, then by 1 again, then a jump by 3, followed by an increase by 1, and finally a decrease by 3.

- ​If we continue this observed pattern, the next difference in the sequence (after the 6) would be 

`6+1=7`

- Using the predicted difference, the next term in the sequence is calculated by adding this difference to the last term in the original sequence:

`36+7=43`

**So, 43 is the predicted next term in the sequence**

**Reply by Sriansh-raj** [reply]
@mitchiemt11  Demn vro you took it so elaborative 👏🏻 👏🏻 

Well and you found a sequence also but There's one more approach ---- each number is calculated as the previous number plus the number of letters when the previous number is spelt.

Now try to get the answer. 😃

**Reply by mitchiemt11** [reply]
Hahaha I had gone too far with it @fabricio-fn 😄 😄

**Comment by mitchiemt11** [comment, marked as answer]
@Sriansh-raj now using that approach, we can break it down in simple terms:

Starting with the number 1:

```
Spelled out: "one"
Number of letters: 3
Next term: 1+3=4
```




Using the number 4:
```
Spelled out: "four"
Number of letters: 4
Next term: 4+4=8
```



Using the number 8:
```
Spelled out: "eight"
Number of letters: 5
Next term: 8+5=13
```

Using the number 13:

```
Spelled out: "thirteen"
Number of letters: 8
Next term: 13+8=21
```


Using the number 21:

```
Spelled out: "twenty-one"
Number of letters: 9
Next term: 21+9=30
```


Using the number 30:

```
Spelled out: "thirty"
Number of letters: 6
Next term: 30+6=36
```

Using the number 36:

```
Spelled out: "thirty-six"
Number of letters: 9
Next term: 36+9=45
```

So
Starting with the number 1, the sequence is generated as:

`1,4,8,13,21,30,36,45`

The next term after 36 is **45.**

**Reply by Sriansh-raj** [reply]
Yeah you got this 👍🏻
